app/views/admin/contest_groups/edit.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Редактиране на група</h1>
+<br />
+<%= form_for [:admin, @contest_group] do |f| -%>
+    <p>
+      <%= f.error_messages %>
+    </p>
+    <%= render 'contest_group_fields', f: f %>
+    <br />
+    <p>
+      <%= f.submit "Обновяване" %><%= link_to "Отказ", admin_contest_groups_path %>
+    </p>
```

Comment:
Please use 2 spaces for indentation. This is the convention that I try to use in the project.